runtime/oti/j9nonbuilder.h

```diff
@@ -2954,6 +2954,9 @@ typedef struct J9Class {
 #define J9INTERFACECLASS_ITABLEMETHODCOUNT(clazz) ((clazz)->totalInstanceSize)
 #define J9INTERFACECLASS_SET_ITABLEMETHODCOUNT(clazz, value) (clazz)->totalInstanceSize = (value)
 
+#define J9ARRAYCLASS_SET_STRIDE(clazz, strideLength) (clazz->flattenedClassCache) = (J9FlattenedClassCache*)(UDATA) strideLength
```

Comment:
Last nitpick (hopefully) - macro parameters should be bracketed in the macro definitions.

`(clazz)->flattenedClassCache`